normandy/health/checks.py

```diff
@@ -21,5 +22,31 @@ def database_connected(app_configs, **kwargs):
     return errors
 
 
+def migrations_applied(app_configs, **kwargs):
+    from django.db.migrations.loader import MigrationLoader
+    errors = []
+
+    # Load migrations from disk/DB
+    loader = MigrationLoader(connection, ignore_no_migrations=True)
```

Comment:
Interestingly, instead of getting the graph of migrations, [Django generates a "migration plan"](https://github.com/django/django/blob/1845bc1d1007751a7f65c66aeddc35f032f6bf41/django/core/management/base.py#L430) to get a list of unapplied migrations.

The migration API is sufficiently undocumented that I have no opinion on which method is better. I'm curious, did you choose your method specifically for some reason?